doc/classes/PhysicsBody2D.xml

```diff
@@ -28,11 +28,14 @@
 			<param index="0" name="distance" type="Vector2" />
 			<param index="1" name="test_only" type="bool" default="false" />
 			<param index="2" name="safe_margin" type="float" default="0.08" />
+			<param index="3" name="recovery_as_collision" type="bool" default="false" />
 			<description>
 				Moves the body along the vector [param distance]. In order to be frame rate independent in [method Node._physics_process] or [method Node._process], [param distance] should be computed using [code]delta[/code].
 				Returns a [KinematicCollision2D], which contains information about the collision when stopped, or when touching another body along the motion.
 				If [param test_only] is [code]true[/code], the body does not move but the would-be collision information is given.
 				[param safe_margin] is the extra margin used for collision recovery (see [member CharacterBody2D.safe_margin] for more details).
+				[param recovery_as_collision] by default ([code]false[/code]), a collision is only reported if the recovery is greater than [param safe_margin], this is the desired behaviour in most cases, having a collision only if there is a minimum velocity,
+				if you want to retrieve collision when the velocity is equals to zero or very low, you need to turn this parameter to ([code]true[/code]).
```

Comment:
I don't know if these are the exact conditions, but the description of the parameter should be of the form `a collision is also reported if` rather than `only reported if`. Also, "is equals to zero" should be either "equals zero" or "is equal to zero".